src/main/java/no/nav/tag/sykefravarsstatistikk/api/altinn/AltinnClient.java

```diff
@@ -63,7 +64,35 @@ public AltinnClient(
             ).getBody());
 
             if (respons.isPresent()) {
-                return respons.get().stream().collect(Collectors.toList());
+                return respons.get();
+            } else {
+                throw new AltinnException("Feil ved kall til Altinn. Response body er null.");
+            }
+
+        } catch (RestClientException e) {
+            log.error("Feil ved kall til Altinn", e);
```

Comment:
Dette gjør kanskje at feilen logges to ganger?

Reply:
Det stemmer, har fjernet den